### PR TITLE
Add linter

### PR DESCRIPTION
**Pull request by 5neepy**

### CODE REVIEW

File: rr_light/reflection_refraction.py [hunk 1]
```diff
@@ -141,12 +143,10 @@ def matplotlib_graph(p1: float, p2: float, p3: float, n1: float, n2: float) -> s
     plt.gca().axes.get_yaxis().set_ticks([])
     
     # Incident beam
-    dc1 = [p1[0], p1[1]]
-    plt.arrow(x=DISTANCE*p1[0], y=DISTANCE*p1[1], dx=DISTANCE*-dc1[0], dy=DISTANCE*-dc1[1], facecolor=COLOR_OF_INCIDENT_BEAM, width=0.4, head_width=1, head_length=1.5, length_includes_head=True)
+    plt.arrow(x=DISTANCE*p1[0], y=DISTANCE*p1[1], dx=DISTANCE*-p1[0], dy=DISTANCE*-p1[1], facecolor=COLOR_OF_INCIDENT_BEAM, width=0.4, head_width=1, head_length=1.5, length_includes_head=True)
```
Comment (5neepy) [inline rr_light/reflection_refraction.py]: Why I have a linter error here? It's not a var

**Comment by ptrtmv** [comment]
please rebase on `main`

File: rr_light/reflection_refraction.py (hunk 1 above)
Comment (ptrtmv) [reply rr_light/reflection_refraction.py]: You have typed `p1: float` and the other points wrongly in the method definition:

`def matplotlib_graph(p1: float, p2: float, p3: float, n1: float, n2: float)` 

and it should be

`def matplotlib_graph(p1: List[float], p2: List[float], p3: List[float], n1: float, n2: float)`

**Review by ptrtmv — CHANGES_REQUESTED** [review]

File: rr_light/reflection_refraction.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 from distutils.util import strtobool
 import math
 from matplotlib import pyplot as plt
-import typing
+from typing import List, Optional
+
+from matplotlib.patches import FancyArrow
```
Comment (ptrtmv) [inline rr_light/reflection_refraction.py]: If you don't use this, please remove it.

**Review by ptrtmv — DISMISSED** [review]

File: rr_light/reflection_refraction.py (hunk 1 above)
Comment (5neepy) [reply rr_light/reflection_refraction.py]: oi

**Comment by 5neepy** [comment]
> please rebase on `main`

image

File: rr_light/reflection_refraction.py (hunk 2 above)
Comment (5neepy) [reply rr_light/reflection_refraction.py]: It's removed

**Comment by ptrtmv** [comment]
> > please rebase on `main`
> 
> image

`git checkout main`
`git pull`
`git checkout  add_linter`
`git rebase main`
`git push -f`

**Pull request merged**